luminosityleds/browser
# add favicon and update metadata

## PR Description



## What type of PR is this? (check all applicable)


## Why is this PR important?

add favicon and update metadata

## Related Tickets & Documents

n/a lol 

## Screenshots



![image](https://github.com/user-attachments/assets/e6fa09c5-9710-48eb-92fd-fe3c8794ed33)



## How to test

1. Run `npm run dev`
2. Go to localhost:3000/
3. Verify that the LL favicon and `Luminosity LEDs` appears in the browser tab


## Added to documentation?


## [optional] Are there any post-deployment tasks we need to perform?


## [optional] What gif best describes this PR?


https://tenor.com/view/coding-is-tough-coding-programming-programmer-quixy-gif-23574044
